pkg/operator/ceph/file/controller.go

```diff
@@ -121,10 +127,22 @@ func (c *FilesystemController) onAdd(obj interface{}) {
 			return
 		}
 	}
+	filesystem, err = updateCephFilesystemStatus(filesystem, filesystemProcessingStatus, c.context)
+	if err != nil {
+		logger.Errorf("Failed to update filesystem %s with status %v: %+v", filesystem.Name, filesystemProcessingStatus, err)
```

Comment:
Instead of logging the error after the method returns, how about logging the error inside the method and not returning the err? This will keep the code a lot simpler. See the `updateClusterStatus()` method for an example.

Reply:
done